tests/xapi-plugins/test_updater.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+class TestUpdate:
+    def test_check_update(self, host):
+        host.call_plugin('updater.py', 'check_update')
+
+    def test_update(self, host):
+        host.yum_save_state()
+        host.call_plugin('updater.py', 'update')
+
+        res = host.call_plugin('updater.py', 'check_update')
+        assert res == "[]"
+
+        host.yum_restore_saved_state()
```

Comment:
Does yum_restore_saved_state still work when no changes have been made to the host?

Reply:
Good question